### PR TITLE
Adds methods: exec() and run()

### PR DESCRIPTION
**Pull request by cognitom**
- `exec()` is just a shortcut of `childProcess.spawn()`
- `run()` returns a promise, and waits until PhantomJS get ready

To use `phantomjs-prebuilt` with WebDriver, we needed to detect when PhantomJS gets ready. But that was little bit messy, like this: 

```javascript
new Promise((resolve, reject) => {
  const program = spawn(phantomjs.path, ['--webdriver=4444'])
  let isFirst = true
  program.stdout.on('data', data => {
    // This detects PhantomJS instance get ready.
    if (!isFirst) return
    isFirst = false
    resolve(program)
  })
}).then(program => {
  // do something
  program.kill()
})
```

By this PR, it'll be easy and readable:

```javascript
phantomjs.run('--webdriver=4444').then(program => {
  // do something
  program.kill()
})
```

If you like this PR, I'd like to add some README text, too. Thanks.

### CODE REVIEW

**Comment by nicks** [comment]
cool! i like this! 

can you fill out a CLA real quick?
https://github.com/Medium/opensource/blob/master/sign-cla.md

File: lib/phantomjs.js [hunk 1]
```diff
@@ -61,3 +63,34 @@ if (exports.path) {
...
+ * @example
+ * // handy with WebDriver
+ * phantomjs.run('--webdriver=4444').then(program => {
+ *   // do something
+ *   program.kill()
+ * })
+ * @returns {Promise} the process of PhantomJS
+ */
+exports.run = function () {
+  var program = exports.exec.apply(null, arguments)
```
Comment (nicks) [inline lib/phantomjs.js]: what happens right now if you pass bogus arguments to the binary? does it resolve or just hang?

Comment (cognitom) [inline lib/phantomjs.js]: Yeah, for example, `--bogus` makes it hang (without error messages). Is it better to be guarded [like this](https://github.com/cognitom/gulp-sketch/blob/master/coffee/index.coffee#L44-L55)?
http://phantomjs.org/api/command-line.html

**Comment by cognitom** [comment]
@nicks sure! Will do.

File: lib/phantomjs.js (hunk 1 above)
Comment (nicks) [inline lib/phantomjs.js]: oh, i was just imagining that we'd reject the promise if the process existed before the promise resolved.

Comment (nicks) [inline lib/phantomjs.js]: oops, meant to say "if the process _exited_ before the promise resolved". if it's not easy to do, don't worry about it.

Comment (cognitom) [inline lib/phantomjs.js]: Oh, I see. I'll have a look. Wait a bit. Thanks.

**Comment by cognitom** [comment]
@nicks the logic is wrapped by `try` block now 😄
Could you have a look?

**Comment by nicks** [comment]
thanks!

**Pull request merged**